src/EnergyPlus/AirflowNetwork/include/AirflowNetwork/Solver.hpp

```diff
@@ -70,33 +70,77 @@ namespace AirflowNetwork {
         Real64 viscosity{AIRDYNAMICVISCOSITY(20.0)};
     };
 
+    // Forward declaration
+    struct AirflowElement;
+
     struct Solver
     {
+        Solver() : PB(0.0)
+        {}
+
+        void allocate();
+        void initialize();
+        void setsky();
+        void airmov();
+        void solvzp(int &ITER);  // number of iterations
+        void filjac(int const NNZE,  // number of nonzero entries in the "AU" array.
+                    bool const LFLAG // if = 1, use laminar relationship (initialization).
+        );
+
+        void clear()
+        {
+            PB = 0.0;
+            //LIST = 0;
+            elements.clear();
+            compnum.clear();
+            properties.clear();
+            AFECTL.deallocate();
+            AFLOW2.deallocate();
+            AFLOW.deallocate();
+            PS.deallocate();
+            PW.deallocate();
+            SUMAF.deallocate();
+            PZ.deallocate();
+            ID.deallocate();
+            IK.deallocate();
+            AD.deallocate();
+            AU.deallocate();
+
+#ifdef SKYLINE_MATRIX_REMOVE_ZERO_COLUMNS
+            newIK.deallocate();
+            newAU.deallocate();
+#endif
+            SUMF.deallocate();
+        }
+
+        std::unordered_map<std::string, AirflowElement *> elements;
+        std::unordered_map<std::string, int> compnum; // Stopgap until all the introspection is dealt with
+
         std::vector<AirProperties> properties;
 
-        int NetworkNumOfLinks;
-        int NetworkNumOfNodes;
+        //int NetworkNumOfLinks;
+        //int NetworkNumOfNodes;
 
-        int const NrInt; // Number of intervals for a large opening
+        //int const NrInt; // Number of intervals for a large opening
 
         // Common block AFEDAT
-        Array1D<Real64> AFECTL;
+        Array1D<Real64> AFECTL; // This gets used in calculate, encapsulation fail
         Array1D<Real64> AFLOW2;
-        Array1D<Real64> AFLOW;
+        Array1D<Real64> AFLOW; // This gets used in calculate, encapsulation fail
         Array1D<Real64> PS;
         Array1D<Real64> PW;
 
         // Common block CONTRL
         Real64 PB;
-        int LIST;
+        //int LIST;
 
         // Common block ZONL
-        // extern Array1D<Real64> RHOZ;
-        // extern Array1D<Real64> SQRTDZ;
-        // extern Array1D<Real64> VISCZ;
+        //Array1D<Real64> RHOZ;
+        //Array1D<Real64> SQRTDZ;
+        //Array1D<Real64> VISCZ;
         Array1D<Real64> SUMAF;
-        // extern Array1D<Real64> TZ; // Temperature [C]
-        // extern Array1D<Real64> WZ; // Humidity ratio [kg/kg]
+        //Array1D<Real64> TZ; // Temperature [C]
+        //Array1D<Real64> WZ; // Humidity ratio [kg/kg]
```

Comment:
A few more to delete.

Reply:
These are the remnants of the old setup before I introduced the properties array. In light of what's going on with the 10x project, it may have been a mistake to collect all these up this way. So I'm leaving these here a placeholder for some performance testing that needs to happen at some point.